modules/storage/flexible-postgresql-server/README.md

```diff
@@ -0,0 +1,77 @@
+# Azure Flexible PostGres-SQL
+
+This Bicep module creates a PostgreSQL Server with zone-redundancy, virtual network access.
```

Comment:
Is the summary and description below still true? The vnet and zone-redundancy resources are commented out in `main.bicep`.

Reply:
If we are not going to implement vnet support in the PR, it should not close https://github.com/Azure/bicep-registry-modules/issues/302.